packages/components/CHANGELOG.md

```diff
@@ -490,7 +629,7 @@
   - Adds `sortingFunction` support in `@columns` declaration
   - Adds `sortedMessageText` support for custom sorting message
   - Updates the `aria-sort` to fallback to "none" instead of "null" (per spec)
-  - Adds support for the `{{each}}` loop's `key` to be defined with `identityKey` (optional; falls back to the Ember default if none is provided)
+  - Adds support for the `\{\{each\}\}` loop's `key` to be defined with `identityKey` (optional; falls back to the Ember default if none is provided)
```

Comment:
Do we need to note somewhere or update the authoring guide to remind ourselves to escape curly braces?

Reply:
there may be other things to escape, so we will see immediately (I hope) if something breaks and fix it accordingly
let's keep things simple for now, there is already a lot of new documentation/guidance for the devs